CefSharp.Example/CefExample.cs

```diff
@@ -188,7 +193,7 @@ public static void Init(AbstractCefSettings settings, IBrowserProcessHandler bro
             settings.RegisterScheme(new CefCustomScheme
             {
                 SchemeName = "localfolder",
-                SchemeHandlerFactory = new FolderSchemeHandlerFactory(rootFolder: DebuggingSubProcess ? @"..\..\..\..\CefSharp.Example\Resources" : "Resources",
```

Comment:
Isn't this one of our changes? I wonder if it's necessary to keep this for some kind of debugging? But if not then this is ok.

Reply:
I rolled back this change, better to keep it as it was